.gitignore

```diff
@@ -1,23 +1,23 @@
-# This file is generated by ModuleSync, do not edit.
-*.iml
 .*.sw[op]
-.DS_Store
```

Comment:
DS_Store is no longer in gitignore.  Is this intentional?

Reply:
Unintentional - Good catch - PR for fix to template https://github.com/puppetlabs/pdk-templates/pull/15
I'll fix this in the PR also.